defaults/main.yml

```diff
@@ -2,6 +2,7 @@
 mariadb_version: "10.4"
 mariadb_distro: "centos{{ ansible_distribution_major_version }}-amd64"
 mariadb_repo_url: "http://yum.mariadb.org/{{ mariadb_version }}/{{ mariadb_distro }}"
+mariadb_suse_repo_url: "https://yum.mariadb.org/10.5/sles/15/x86_64"
```

Comment:
to powinno być przepisane, żeby zmienna mariadb_repo_url była ustawiana w zależności od systemu i wersji w jakimś miejscu a nie dedykowana zmienna dla suse